src/Globals.cpp

```diff
@@ -49,5 +50,6 @@ std::atomic<bool> AUTONOMOUS = false;
 robot::types::mountedperipheral_t mountedPeripheral = robot::types::mountedperipheral_t::none;
 control::PlanarArmController<2> planarArmController(createArmKinematics(),
 													Constants::arm::SAFETY_FACTOR);
+control::SwerveController swerveController(Constants::EFF_WHEEL_BASE, Constants::ROBOT_LENGTH);
```

Comment:
/We should use the real wheel base instead of the effective wheel base. If `ROBOT_WIDTH` isn't already a constant, we should add it